tensorflow_probability/python/experimental/mcmc/covariance_reducer.py

```diff
@@ -41,9 +41,23 @@
     'CovarianceReducerState', 'init_structure, cov_state')
 
 
+def _get_sample(current_state, kernel_results):
+  del kernel_results
+  return current_state
+
+
 class CovarianceReducer(reducer_base.Reducer):
   """`Reducer` that computes a running covariance.
 
+  `CovarianceReducer` computes a running covariance on samples evaluated on
```

Comment:
Around here and in the method docstring, say what the default means (i.e., covariance directly over the latent state, ignoring kernel results).